cmd/notifyStale.go

```diff
@@ -83,9 +82,12 @@ var notifyStaleCmd = &cobra.Command{
 				}
 
 				if shouldSendMessage {
-					message := fmt.Sprintf("The following pull request has no activity from a Chia team member in the last 7 days: %s", pr.URL)
+					status := "firing"
+					title := "The following pull request has no activity from a Chia team member in the last 7 days:"
+					description := pr.URL
```

Comment:
I thought URL is usually the API's URL, and the HTML URL element is actually what a human reader would expect to get?